exampleCourse/questions/demo/drawing/resistorCapacitorCircuit/question.html

```diff
@@ -0,0 +1,32 @@
+<pl-question-panel>
+
+    <p>
+        <pl-drawing width="440" height="260"  grid-size="0" answers-name="answer">
+            <pl-drawing-initial>
+                <pl-battery        x1={{params.pA.0}} y1={{params.pA.1}} x2={{params.pB.0}} y2={{params.pB.1}} label="{{params.V}} V"></pl-battery>
```

Comment:
Unless there's a reason for all the spaces after the element name, I'd remove them.